<ui version="4.0" >
 <class>CSceneControllerGuiComp</class>
 <widget class="QWidget" name="CSceneControllerGuiComp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>51</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>51</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="toolTip" >
   <string>Display navigation bar</string>
  </property>
  <property name="styleSheet" >
   <string notr="true" />
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="styleSheet" >
      <string notr="true" >QFrame, QToolButton {
     border: 1px solid #8f8f91;
     border-radius: 14px;

 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

QWidget{
}

QToolButton {
     border: 1px solid #8f8f91;
     border-radius: 3px;
 }

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }
</string>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QToolButton" name="ResetViewButton" >
        <property name="toolTip" >
         <string>Reset display position</string>
        </property>
        <property name="styleSheet" >
         <string notr="true" />
        </property>
        <property name="text" >
         <string>...</string>
        </property>
        <property name="icon" >
         <iconset resource="../iqtgui/iqtgui.qrc" >
          <normaloff>:/Icons/ZoomReset</normaloff>:/Icons/ZoomReset</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="ZoomInButton" >
        <property name="toolTip" >
         <string>Zoom in</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../iqtgui/iqtgui.qrc" >
          <normaloff>:/Icons/ZoomIn</normaloff>:/Icons/ZoomIn</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="ZoomOutButton" >
        <property name="toolTip" >
         <string>Zoom out</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../iqtgui/iqtgui.qrc" >
          <normaloff>:/Icons/ZoomOut</normaloff>:/Icons/ZoomOut</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="RotateLeftButton" >
        <property name="toolTip" >
         <string>Rotate left</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../iqtgui/iqtgui.qrc" >
          <normaloff>:/Icons/RotateLeft.svg</normaloff>:/Icons/RotateLeft.svg</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRepeat" >
         <bool>true</bool>
        </property>
        <property name="autoRepeatDelay" >
         <number>300</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="RotateRightButton" >
        <property name="toolTip" >
         <string>Rotate right</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../iqtgui/iqtgui.qrc" >
          <normaloff>:/Icons/RotateRight.svg</normaloff>:/Icons/RotateRight.svg</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRepeat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../iqtgui/iqtgui.qrc" />
  <include location="../iqtgui/iqtgui.qrc" />
 </resources>
 <connections/>
</ui>
